geomstats/geometry/special_euclidean.py

```diff
@@ -238,6 +263,34 @@ def regularize_tangent_vec(
 
         return regularized_vec
 
+    def matrix_from_vector(self, vec):  # TODO Fails with Tensorflow for now
```

Comment:
It can probably be clear from the tests that this fails with TF: the corresponding test will have the decorator @...only_np_and_pytorch

so no need for this comment.

Reply:
update?